docs/Using Fleet/Puppet-module.md

```diff
@@ -0,0 +1,152 @@
+# Puppet module
+
+_Available in Fleet Premium_
+
+Use [Fleet's Puppet module](https://forge.puppet.com/modules/fleetdm/fleetdm/readme) to automatically install custom configuration profiles on your macOS hosts based on host attributes you define in Puppet.
+
+The module also includes functions for releasing a macOS host from await configuration and sending any custom MDM commands.
+
+## Setup
+
+To setup the Puppet module, we will do the following steps:
```

Comment:
there's a detail here that I think it's implicit, pointing out just in case: you need to [install](https://www.puppet.com/docs/puppet/8/modules_installing.html) Fleet's Puppet module before doing any of this.

Reply:
Ah, thanks Roberto! I added an "install the Puppet module" step.

We want to be explicit in the docs and assume our users are following the instructions step by step.